clang/lib/CodeGen/CGCall.cpp

```diff
@@ -2569,6 +2569,18 @@ void CodeGenModule::ConstructAttributeList(StringRef Name,
 
     if (TargetDecl->hasAttr<ArmLocallyStreamingAttr>())
       FuncAttrs.addAttribute("aarch64_pstate_sm_body");
+
+    if (auto *ModularFormat = TargetDecl->getAttr<ModularFormatAttr>()) {
+      // TODO: Error checking
```

Comment:
This is a heck of a TODO :)   Though, I'd expect us to do diagnostics during our normal checking of the format string, so we shouldn't really require anything here.

Reply:
Hah, fair; this is very much a Draft PR. My intent was to get this in front of a bunch of eyes sooner rather than later, as this PR set touches everything every layer from the compiler through to libc (skipping the linker).

Reply:
Just another ping here, I saw a little bit of error checking above, but probably this comment should disappear/that should be fleshed out.

Reply:
Ah thanks; pretty much all error stuff should happen much earlier than this, so I think there's nothing more to do here. Removed the comment.